### PR TITLE
fix: Replace applymap with map for pd.DataFrame

### PR DESCRIPTION
**Pull request by esloch**

### CODE REVIEW

File: pysus/data/__init__.py [hunk 1]
```diff
@@ -109,7 +109,8 @@ def dbf_to_parquet(dbf: str, _pbar=None) -> str:
                 _pbar.update(chunk_size)
 
             chunk_df = pd.DataFrame(chunk)
-            table = pa.Table.from_pandas(chunk_df.applymap(decode_column))
+            #breakpoint()
```
Comment (luabida) [inline pysus/data/__init__.py:112]: I think this line could be removed completely

**Review by luabida — COMMENTED** [review]
I see some commented code in the PR. Is there any reason to keep them in the repo?

File: pysus/online_data/IBGE.py [hunk 2]
```diff
@@ -97,6 +97,12 @@ def get_sidra_table(
         query += f'/h/{headers}'
 
     url = base_url + query
+
+    #data = requests.get(url)
```
[the remark this answers is not in the document]
Comment (esloch) [reply pysus/online_data/IBGE.py:101]: I was testing these changes for a previous issue, I just uploaded it, I will remove it.

**Comment by esloch** [comment]
> I see some commented code in the PR. Is there any reason to keep them in the repo?

The commented codes are here because this PR has not yet been finalized, I still have to verify that we have tests for these functions, for this reason they still remain.

**Review by fccoelho — APPROVED** [review]

**Pull request merged**